Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs

```diff
@@ -228,6 +227,11 @@ private void ResetButtonStyle()
             ButtonStyle = ButtonStyle.Standalone;
         }
 
+        [DefaultValue(false), Description("Always use the set text, even if the 'DialogResult' value has been changed. (Note: You can manage the 'DialogResult' string values via the KryptonManager on the parent KryptonForm.)")]
```

Comment:
This should be true, to be compatible with existing iplementations..
i.e. a DefaultOK result for a button that the dev has set to be `Commit`, should still say Commit after updating to this dll.

Currently it will state something different, as therefore translations etc will also be messed up.

Reply:
Fixed